shared/pub/metadata.js

```diff
@@ -49,3 +49,49 @@ export const getTextAbstract = (docJson) => {
 	}
 	return abstract;
 };
+
+export const getGSNotes = (notes) => {
+	return notes.reduce((unique, note) => {
+		const noteArray = [];
+		const noteType = note.json[0].type;
+		const noteTypes = {
```

Comment:
Nit: this can be factored out of the function.

Reply:
I think this is what you meant?